pkg/account/component_test.go

```diff
@@ -88,3 +96,254 @@ func TestUpdatePasswordWrongPwd(t *testing.T) {
 	assert.True(t, err != nil)
 
 }
+
+func TestUpdateAccount(t *testing.T) {
+	var mockCtrl = gomock.NewController(t)
+	defer mockCtrl.Finish()
+	mockKeycloakClient := mock.NewAccKeycloakClient(mockCtrl)
+	mockEventDBModule := mock.NewEventsDBModule(mockCtrl)
+	var mockLogger = log.NewNopLogger()
+
+	var accountComponent = NewComponent(mockKeycloakClient, mockEventDBModule, mockLogger)
+
+	accessToken := "access token"
+	realmName := "master"
+	userID := "123-456-789"
+	username := "username"
+	ctx := context.WithValue(context.Background(), cs.CtContextAccessToken, accessToken)
+	ctx = context.WithValue(ctx, cs.CtContextRealm, realmName)
+	ctx = context.WithValue(ctx, cs.CtContextUserID, userID)
+	ctx = context.WithValue(ctx, cs.CtContextUsername, username)
+
+	var id = "1234-7454-4516"
+	//var username = "username"
+	var email = "toto@elca.ch"
+	var enabled = true
+	var emailVerified = true
+	var firstName = "Titi"
+	var lastName = "Tutu"
+	var phoneNumber = "+41789456"
+	var phoneNumberVerified = true
+	var label = "Label"
+	var gender = "M"
+	var birthDate = "01/01/1988"
```

Comment:
Why 1988 ? 😄

Reply:
People are not trustworthy, they have the tendency to leak information related to them :D

Reply:
Just a wonderful year ;)